lib/support/tcpdump.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const { spawn } = require('child_process');
+const get = require('lodash.get');
+const mkdirp = require('mkdirp');
+
+/**
+ * Create a new TCPDUMP instance
+ * @class
+ */
+class TCPDUMP {
+  constructor(directory, options, iteration) {
+    this.directory = directory;
+    this.options = options;
+    this.iteration = iteration;
+  }
+
+  start() {
+    const useTcpDump = get(this.options, 'pcap', false);
+    if (useTcpDump === true || useTcpDump === 'true') {
+      // Create Capture dir
```

Comment:
Add a info log here and when we stop, it's perfectly ok I think, then it's so much easier to follow the flow what happens.